UsingSeal.md

```diff
@@ -254,49 +262,147 @@ Start by generating a master seed for the key server.
 
+```
+2025-06-15T02:02:56.303459Z  INFO key_server: Unassigned derived public key with index 0: "$PUBKEY_0"
+2025-06-15T02:02:56.303957Z  INFO key_server: Unassigned derived public key with index 1: "$PUBKEY_1"
+2025-06-15T02:02:56.304418Z  INFO key_server: Unassigned derived public key with index 2: "$PUBKEY_2"
+```
+
 Register the first client:
-- Use the first unassigned derived public key (with derivation index 0).
-- Register a new key server on-chain by calling the `create_and_transfer_v1` function from the `seal::key_server` module (see Open Mode above for usage).
-- In your key server configuration file:
-  - Add a new client entry.
+
+- Register a new key server on-chain by calling the `create_and_transfer_v1` function from the `seal::key_server` module with the first unassigned derived public key (with derivation index 0) $PUBKEY_0.
+
+```shell
+sui client call --function create_and_transfer_v1 --module key_server --package 0x62c79dfeb0a2ca8c308a56bde530ccf3846535e1623949d45c90d23128afff52 --args $YOUR_SERVER_NAME https://your-url-here 0 $PUBKEY_0 --gas-budget 10000000
+# outputs $KEY_SERVER_OBJECT_ID_0
+```
+
+
+- Add an entry in config file:
   - Set `client_master_key` to type `Derived` with `derivation_index` as 0. 
+  - Set `$KEY_SERVER_OBJECT_ID_0` from the output above. 
   - Include the list of packages this client will use.
-Restart the key server to apply the changes, using `cargo run --bin key-server`.
 
-To add more clients:
-- Check the key server logs for the next unassigned derived public key.
-- Register the new public key on-chain using `create_and_transfer_v1`.
-- Add a new client configuration using the next `derivation_index` value and corresponding packages.
+For example: 
+
+```yaml
+    - name: "alice"
+      client_master_key: !Derived
+        derivation_index: 0
+      key_server_object_id: "$KEY_SERVER_OBJECT_ID_0"
+      package_ids:
+        - "0x1111111111111111111111111111111111111111111111111111111111111111"
+```
+
+- Restart the key server to apply the config changes.
+
+```shell
+MASTER_SEED=$MASTER_SEED CONFIG_PATH=crates/key-server/key-server-config.yaml cargo run --bin key-server 
+```
+Or with Docker: 
+```
+docker run -p 2024:2024 \
+  -v $(pwd)/crates/key-server/key-server-config.yaml:/config/key-server-config.yaml \
+  -e CONFIG_PATH=/config/key-server-config.yaml \
+  -e MASTER_SEED=$MASTER_SEED \
+  seal-key-server
+```
+To add more clients, repeat the above steps with unassigned public keys, e.g. e.g `$PUBKEY_1, $PUBKEY_2`.
 
 #### Export / Import Keys
 
 In rare cases where you need to export a client key:
 - Use the `seal-cli` tool as `cargo run --bin seal-cli derive-key --seed $MASTER_SEED --index X`. 
 Replace `X` with the `derivation_index` of the key you want to export.
 The tool will output the corresponding master key, which can be imported by another key server if needed.
+
+```shell
+cargo run --bin seal-cli derive-key --seed $MASTER_SEED --index 0
+Master key: $CLIENT_MASTER_KEY
```

Comment:
I think that $CLIENT_MASTER_KEY and $CLIENT_MASTER_PUBKEY are confusing, let's have example output?

Reply:
this is easier to read than an example value, so now i can search for CLIENT_MASTER_PUBKEY and understand where this value is used in the command later. I use <value> instead of $value now, since that's what sui docs now use as convention.